<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(244, 249, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="mainWindowFrame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>600</width>
      <height>400</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: qlineargradient(
    x1: 0, y1: 0, x2: 1, y2: 1,
    stop: 0 rgba(196, 236, 236, 1), stop: 1 rgba(255, 255, 255, 1)
);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="definitionsButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>0</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/defi_light_mode.png</normaloff>../images/defi_light_mode.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="recordingLabel">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>150</y>
       <width>150</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 9pt &quot;Arial&quot;;
background: none;
border: none;</string>
     </property>
     <property name="text">
      <string>Status: Not recording</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>264</y>
       <width>500</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: black;
background-color: black;
</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>278</y>
       <width>500</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: black;
background-color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>292</y>
       <width>500</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: black;
background-color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>306</y>
       <width>500</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: black;
background-color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>250</y>
       <width>500</width>
       <height>1</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: black;
background-color: black;</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="microphoneButton">
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>70</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/microphoneOff.png</normaloff>../images/microphoneOff.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>120</width>
       <height>120</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="logoButton">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>10</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/icon.png</normaloff>../images/icon.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>70</width>
       <height>70</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="gClave">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>232</y>
       <width>50</width>
       <height>95</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../images/gClave.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clearSheetButton">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>210</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <underline>true</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;
font: 13pt &quot;Arial&quot;;
</string>
     </property>
     <property name="text">
      <string>Clear Sheet</string>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="exportButton">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>330</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>13</pointsize>
       <italic>false</italic>
       <bold>false</bold>
       <underline>false</underline>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 13pt &quot;Arial&quot;;
color: white; 
background: qradialgradient(
    cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
    radius: 1.35, stop: 0 rgba(35, 120, 99, 1), stop: 1 rgba(77, 206, 175, 1)
);

border-radius: 10%;</string>
     </property>
     <property name="text">
      <string>Export Sheet</string>
     </property>
     <property name="iconSize">
      <size>
       <width>25</width>
       <height>25</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="startRecordingLabel">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>50</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
font: 13pt &quot;Arial&quot;;
background: none;
border: none;</string>
     </property>
     <property name="text">
      <string>Press to start recording</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pauseButton">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>200</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/pauseee.png</normaloff>../images/pauseee.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="continueButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>200</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/playyy.png</normaloff>../images/playyy.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="backButtonMainWindow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>50</width>
       <height>49</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/back_light.png</normaloff>../images/back_light.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <widget class="QPushButton" name="stopButton">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>200</y>
       <width>40</width>
       <height>40</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <underline>false</underline>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background: none;
border: none;
color: black;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>../images/stopp.png</normaloff>../images/stopp.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>30</width>
       <height>30</height>
      </size>
     </property>
    </widget>
    <zorder>line5</zorder>
    <zorder>line4</zorder>
    <zorder>line3</zorder>
    <zorder>line2</zorder>
    <zorder>line1</zorder>
    <zorder>gClave</zorder>
    <zorder>definitionsButton</zorder>
    <zorder>recordingLabel</zorder>
    <zorder>microphoneButton</zorder>
    <zorder>logoButton</zorder>
    <zorder>clearSheetButton</zorder>
    <zorder>exportButton</zorder>
    <zorder>startRecordingLabel</zorder>
    <zorder>pauseButton</zorder>
    <zorder>continueButton</zorder>
    <zorder>backButtonMainWindow</zorder>
    <zorder>stopButton</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
